integration/basic_test.go

```diff
@@ -177,6 +177,48 @@ func TestUpdateCommentQuery(t *testing.T) {
 	assert.Equal(t, expected, res)
 }
 
+func TestDeleteCommentMethod(t *testing.T) {
+	t.Parallel()
+	ctx, collection := setup.Setup(t, shareddata.Scalars)
+	name := collection.Name()
```

Comment:
```suggestion
	name := collection.Database().Name()
```

Otherwise, you are checking that collection name is present in a list of database names.

Please see changes in #953.